src/themes/huraga/html/partial_menu.html.twig

```diff
@@ -1,69 +1,100 @@
+                        {{ 'Wallet'|trans }}
+                    </a>
+                </li>
+            {% endif %}
 
-    {% if settings.side_menu_payments %}
-    <li class="main-menu">
-        <a href="{{ '/client/balance'|link }}"><span class="awe-credit-card"></span>{{ 'Payment History'|trans }}</a>
-    </li>
-    {% endif %}
+            {% if (guest.extension_is_on({"mod":"news"}) and settings.side_menu_news) %}
+                <li class="nav-item offcanvas-item">
+                    <a class="nav-link d-flex align-items-center gap-2" href="{{ '/news'|link }}">
+                        <svg class="svg-icon" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 5L20 19L4 19L4 5H20M20 3H4C2.89 3 2 3.89 2 5V19C2 20.11 2.89 21 4 21H20C21.11 21 22 20.11 22 19V5C22 3.89 21.11 3 20 3M18 15H6V17H18V15M10 7H6V13H10V7M12 9H18V7H12V9M18 11H12V13H18V11Z" /></svg>
+                        {{ 'News'|trans }}
+                    </a>
+                </li>
+            {% endif %}
 
-    {% if (guest.support_kb_enabled and settings.side_menu_kb) %}
-    <li class="main-menu">
-        <a href="{{ '/support/kb'|link }}"><span class="awe-book"></span>{{ 'Knowledge Base'|trans }}</a>
-    </li>
-    {% endif %}
+            {% if (guest.support_kb_enabled() and settings.side_menu_kb) %}
+                <li class="nav-item offcanvas-item">
+                    <a class="nav-link d-flex align-items-center gap-2" href="{{ 'support/kb'|link }}">
+                        <svg class="svg-icon" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 1L14 6V17L19 12.5V1M21 5V18.5C19.9 18.15 18.7 18 17.5 18C15.8 18 13.35 18.65 12 19.5V6C10.55 4.9 8.45 4.5 6.5 4.5C4.55 4.5 2.45 4.9 1 6V20.65C1 20.9 1.25 21.15 1.5 21.15C1.6 21.15 1.65 21.1 1.75 21.1C3.1 20.45 5.05 20 6.5 20C8.45 20 10.55 20.4 12 21.5C13.35 20.65 15.8 20 17.5 20C19.15 20 20.85 20.3 22.25 21.05C22.35 21.1 22.4 21.1 22.5 21.1C22.75 21.1 23 20.85 23 20.6V6C22.4 5.55 21.75 5.25 21 5M10 18.41C8.75 18.09 7.5 18 6.5 18C5.44 18 4.18 18.19 3 18.5V7.13C3.91 6.73 5.14 6.5 6.5 6.5C7.86 6.5 9.09 6.73 10 7.13V18.41Z" /></svg>
+                        {{ 'Knowledge base'|trans }}
+                    </a>
+                </li>
+            {% endif %}
 
-    {% if (guest.extension_is_on({"mod":"news"}) and settings.side_menu_news) %}
-    <li class="main-menu">
-        <a href="{{ '/news'|link }}"><span class="awe-edit"></span>{{ 'News'|trans }}</a>
-    </li>
-    {% endif %}
-</ul>
+            {% if settings.sidebar_balance_enabled and client %}
+                <li class="pt-3 ps-3">
+                    <h5 class="text-secondary d-block pb-1">{{ 'Account balance'|trans }}</h5>
+                    <h4><strong>{{ profile.balance | money(profile.currency) }}</strong></h4>
```

Comment:
For my taste, this section isn't well defined. The font sizes are too different, there's no icons in front of the Account balance, etc. Doesn't have to be changed now, but would be nice to find a more uniform approach

![grafik](https://github.com/FOSSBilling/FOSSBilling/assets/1939311/f94f86e1-52d4-4090-bc36-9ae9b485a86b)

Reply:
I'm sorry your tastes don't like it. Feel free to submit a PR at a later date if you can find a way of doing it that's better